### PR TITLE
Adding support for indexed normals on import #1029

### PR DESCRIPTION
**Pull request by koen-v**
Hello Autodesk, I am re-opening this pull request, I added the test.

Thanks,
Koen

### CODE REVIEW

**Comment by koen-v** [comment]
this is related to the issue:
The normals dont get imported properly when they are indexed #826

**Comment by kxl-adsk** [comment]
Thank you @koen-v. I just started the PF and assigned Jerry to review.

File: lib/mayaUsd/fileio/translators/translatorMesh.cpp [hunk 1]
```diff
@@ -120,6 +120,7 @@ TranslatorMeshRead::TranslatorMeshRead(
     // timeInterval or default.
     VtVec3fArray        points;
     VtVec3fArray        normals;
+    TfToken             normals_interpolation;
```
Comment (JGamache-autodesk) [inline lib/mayaUsd/fileio/translators/translatorMesh.cpp:123]: Very minor, but this snake_case variable does stand out in a sea of camelCase variables. Can you make it normalsInterpolation?

**Review by JGamache-autodesk — APPROVED** [review]

File: lib/mayaUsd/fileio/translators/translatorMesh.cpp (hunk 1 above)
Comment (koen-v) [reply lib/mayaUsd/fileio/translators/translatorMesh.cpp]: sorry about that, changed and committed

**Pull request merged**